src/layout/_page.scss

```diff
@@ -14,3 +14,7 @@
     padding: ms(10) ms(2);
   }
 }
+
+.page-section--compact {
```

Comment:
@robwierzbowski @stewartyu @josephco My opinions are the following:

1. If we want to increase specificity, I like nesting with `&` because I find it more readable. 
1. We don't currently chain our BEM modifiers.
1. @josephco I think your change to element from modifier is much clearer